package.json

```
@@ -25,33 +25,34 @@
     "test:ci": "hardhat test"
   },
   "dependencies": {
-    "@1inch/token-plugins": "1.1.2",
-    "@1inch/solidity-utils": "3.0.1",
-    "@1inch/farming": "3.0.2",
-    "@openzeppelin/contracts": "4.9.2"
+    "@1inch/token-plugins": "https://github.com/1inch/token-plugins.git#feature/bump-solidity",
+    "@1inch/farming": "https://github.com/1inch/farming.git#feature/bump-solidity",
```

```suggestion
    "@1inch/token-plugins": "1.3.0",
    "@1inch/farming": "3.3.0",
```
It should be applied after merging https://github.com/1inch/farming/pull/76